go.mod

```diff
@@ -53,6 +53,7 @@ require (
 	github.com/naoina/toml v0.1.2-0.20170918210437-9fafd6967416
 	github.com/olekukonko/tablewriter v0.0.5
 	github.com/peterh/liner v1.1.1-0.20190123174540-a2c9a5303de7
+	github.com/pion/stun v0.6.1
```

Comment:
Why not use `2.0.0`?

Reply:
I will check why `go get` not get the latest version

Reply:
You probably have to do `go get github.com/pion/stun/v2`